examples/events/globalLocalEventsExample/src/MyButton.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ *  MyButton.cpp
+ *  globalLocalEventsExample
+ *
+ *  Created by Andreas Borg on 10/2/15
+ *  Copyright 2015 Local Projects. All rights reserved.
```

Comment:
I think "all rights reserved" is not an appropriate copyright line, as we then couldn't use/include the code in OF, which is MIT-licensed.